lib/CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ if(CURL_BUILD_TESTING)
       ${CSOURCES} > "${CMAKE_CURRENT_BINARY_DIR}/unitprotos.h"
     DEPENDS "${PROJECT_SOURCE_DIR}/scripts/extract-unit-protos" ${CSOURCES}
     VERBATIM)
-  add_custom_target(curlu-unitprotos ALL DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/unitprotos.h")
+  add_custom_target(curlu-unitprotos DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/unitprotos.h")
```

Comment:
So what triggers this file to get generated?

Reply:
It's hooked up to `units` as a dependency:
https://github.com/curl/curl/blob/952c929bdf70645ee263c45a19518ae976a0292d/tests/unit/CMakeLists.txt#L37-L38